blotztask-ui/src/app/auth/signin/page.tsx

```diff
@@ -53,32 +53,31 @@ const LoginPage = () => {
 
   return (
     <div className="h-full justify-center flex flex-col items-center">
-      <div className="flex flex-col gap-4 bg-white p-5 rounded-lg shadow-md w-96">
-        <h1 className={styles.title}>User Login</h1>
+      <div className="flex flex-col gap-4 bg-white p-5 rounded-lg w-96">
+        <h1 className={styles.title}> Welcome to Blotz</h1>
+        <p className={styles.subtitle}>Enter your email and password to start</p>
```

Comment:
```suggestion
        <h1 className={styles.title}> Welcome to Blotz</h1>
        <p className={styles.subtitle}>Enter your email and password to start</p>
```

try to use tailwind class since our project is using tailwind , you dont need to create class for the element